ramls/oai-pmh-view.raml

```diff
@@ -10,6 +10,8 @@ documentation:
 
 types:
   oaipmhInstances: !include oaipmh/oaipmhinstances.json
+  oaipmhUpdatedInstanceIds: !include oaipmh/oaipmhupdatedinstanceids.json
+  oaipmhEnrichedInstances: !include oaipmh/oaipmhenrichedinstances.json
```

Comment:
This type does not appear to be referenced anywhere in the schema

Reply:
I havce checked and added a new type definitoon for POST "instances ids" with "skip suppressed from discovery" flag. Currently it seems like all the defined types are used.